acceptance-tests/src/test/java/tech/pegasys/web3signer/dsl/utils/DataStructureUtilAdapter.java

```diff
@@ -135,7 +135,7 @@ private ExecutionPayload randomExecutionPayload(final SpecVersion specVersion) {
                     .blockHash(util.randomBytes32())
                     .transactions(util.randomExecutionPayloadTransactions())
                     .withdrawals(() -> randomExecutionPayloadWithdrawals(specVersion))
-                    .excessDataGas(util::randomUInt256));
+                    .excessDataGas(util::randomUInt64));
```

Comment:
Is this change related to the teku upgrade?

Reply:
It is, the excessDataGas field is now uint64 instead of unint256. We only have this for some data in tests so far.